_integration-test/run.sh

```diff
@@ -111,10 +115,9 @@ declare -a EVENT_TEST_STRINGS=(
   '"title":"a failure"'
   '"object":"42"'
 )
-for i in "${EVENT_TEST_STRINGS[@]}"
-do
+for i in "${EVENT_TEST_STRINGS[@]}"; do
   echo "Testing '$i'..."
-  echo "$EVENT_RESPONSE" | grep "$i[,}]" >& /dev/null
+  echo "$EVENT_RESPONSE" | grep "$i[,}]" >&/dev/null
```

Comment:
everything looks good to me now

Just one minor thing, is there a way to disable removing the space here? I actually prefer that, if not it's fine too

Reply:
I don't think so :/